### PR TITLE
[TD-11715]<fix>taosdemo sml unsupport sample

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: src/demoInsert.c [hunk 1]
```diff
@@ -2911,13 +2911,14 @@ int startMultiThreadInsertData(int threads, char *db_name, char *precision,
...
 
     // read sample data from file first
-    int ret;
-    if (stbInfo && stbInfo->iface != SML_IFACE) {
+    int ret = 0;
+    if (stbInfo->iface != SML_IFACE) {
+      if (stbInfo && stbInfo->iface != SML_IFACE) {
         ret = prepareSampleForStb(stbInfo);
-    } else {
+      } else {
```
Comment (sangshuduo) [inline src/demoInsert.c:2918]: indent

File: src/demoInsert.c [hunk 2]
```diff
@@ -3368,8 +3369,10 @@ int insertTestProcess() {
     }
 
     // pretreatment
-    if (prepareSampleData()) {
+    if(g_args.iface != SML_IFACE) {
+      if (prepareSampleData()) {
```
Comment (sangshuduo) [inline src/demoInsert.c:3373]: indent

**Review by sangshuduo — CHANGES_REQUESTED** [review]

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by afwerar — APPROVED** [review]
LGTM

**Pull request merged**